internal/services/spec/validator/utils.go

```diff
@@ -12,22 +12,24 @@ type (
 	utils struct {
 		pathResolver pathResolver
 		document     spec.Document
+		projectDir   string
 	}
 )
 
 func newUtils(
 	pathResolver pathResolver,
 	document spec.Document,
+	projectDir string,
 ) *utils {
 	return &utils{
+		projectDir:   projectDir,
```

Comment:
added projectDir for ability of using flags --arch-file and --project-path contains in the separated dirs.  

for example:
```
go-arch-lint check --project-path /go/proj1 --arch-file /arch/arch.yml  
go-arch-lint check --project-path /go/proj2 --arch-file /arch/arch.yml
```

in current release we can't do that (I think it's bug)